docs/Accessibility.md

```diff
@@ -0,0 +1,15 @@
+# Accessibility
+
+This document outlines the accessibility information for the website.
+
+## Alt-text
+
+All images include alt text, a short description of the image.
+
+## Color
+
+The primary colors used in this website is `#660000`, `#F5F5F5`, `#465052`, `#FFFFFF`, and `#000000`.
```

Comment:
Thoughts on adding something like this https://github.com/olizilla/swatches to include color swatches rather than just color codes?

Reply:
Did not know this was a thing, added now!